metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "puppetlabs-bolt_plugin_helper",
+  "version": "0.1.0",
+  "author": "Puppet, Inc.",
+  "summary": "A helper for writing Bolt plugins",
+  "license": "Apache-2.0",
+  "source": "git@github.com/puppetlabs/puppetlabs-bolt_plugin_helper",
+  "dependencies": [
+
+  ],
+  "operatingsystem_support": [
```

Comment:
Do we get dinged on the forge for not having these? We should talk to them about how this doesn't really apply to all modules these days.

Reply:
Forge doesn't care if this is present as far as I can tell.